api/src/main/java/org/openmrs/api/db/DiagnosisDAO.java

```diff
@@ -21,7 +21,7 @@
  * @since 2.2
  */
 public interface DiagnosisDAO {
```

Comment:
This file seems not to have any real changes, you can just revert these format changes